apis_bibsonomy/models.py

```diff
@@ -29,3 +33,21 @@ def bibtexjson(self):
 
     def get_absolute_url(self):
         return reverse("apis_bibsonomy:referencedetail", kwargs={"pk": self.pk})
+
+    @property
+    def referenced_object(self):
+        return self.content_type.get_object_for_this_type(id=self.object_id)
+
+    @property
+    def similar_references(self, with_self=False):
+        similarity_fields = getattr(
+            settings,
+            "REFERENCE_SIMILARITY",
```

Comment:
I would use a plugin specific prefix for settings. eg `BIBSONOMY_REFERENCE_SIMILARITY`, so its clear where the settings belong